PRS10001-11000/PRS10474Yostos.xml

```diff
@@ -39,19 +39,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
+         <change who="ES" when="2021-12-28">updated details</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <person sex="1">
-               <persName>
-                  
-                  Yosṭos
-               </persName>
-               <occupation> martyr</occupation>
```

Comment:
I did as <textClass>
              <keywords>
                <term key="martyr"></term>
              </keywords>
            </textClass> for other martyrs from the GS